ros2cli/ros2cli/xmlrpc/marshal/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from . import generic
+from . import rclpy
+
+
+__all__ = [
+    'generic',
+    'rclpy'
+]
```

Comment:
Same for this file.

Reply:
Same as above.

Reply:
With https://github.com/ros2/ros2cli/pull/493/files#r413242967 this shouldn't be necessary.

Reply:
Almost, see https://github.com/ros2/ros2cli/pull/493/commits/e923320345bd371c53ca8e01457acfc7ff61bec9. Relative imports are weird, `from . import module` works but `from . import module.submodule` doesn't.